### PR TITLE
INFRA-3620 Update ve build

### PR DESCRIPTION
**Pull request by derickfernando-airship**
I cherry picked edits for review, I'll add the refactor and cleanup commits in a separate PR.

Noteworthy Changes:

1. Everything is now built into ~/apptimize-ve/ave etc.
2. Added linux build using Vagrantfile, example in README.md
3. Upgrade PyPy to latest 2.X series.

### CODE REVIEW

File: os/osx/brew.sh [hunk 1]
```diff
@@ -71,12 +60,12 @@ wget \
...
 zlib
 
-if [ ! -e /Applications/Xcode.app ]; then
+if [ ! -e /Applications/Docker.app ]; then
     $BREW cask install docker
 fi
 
 # test automation stuff
-$BREW install --HEAD libimobiledevice
+# $BREW install --HEAD libimobiledevice # TODO Delete?
```
Comment (derickfernando-airship) [inline os/osx/brew.sh:68]: @honeybeep May I delete this line altogether? I can file a separate ticket if we want to fix this. Thanks,

**Review by bschlief — APPROVED** [review]

File: os/osx/brew.sh (hunk 1 above)
Comment (honeybeep) [reply os/osx/brew.sh:68]: does leaving it in / leaving it in without forcing HEAD not work? if we need to delete it to checkpoint the ve work that's ok, people can always brew install things manually if they're playing with automation.

**Review by honeybeep — COMMENTED** [review]
very excited! i guess next steps are figuring out the docs, and specifically how people are going to pull down these changes. we need to upload to ve@netops?

File: os/osx/brew.sh (hunk 1 above)
Comment (derickfernando-airship) [reply os/osx/brew.sh]: Left it in (it installs fine), we'll find out if it works when I work on the e2e automation scripts.

**Comment by derickfernando-airship** [comment]
I'm going to merge these changes. Opening another PR for refactoring shortly, let's discuss how if these files should be published if at all in that PR.

**Pull request merged**